### PR TITLE
enumerated items import statements

### PR DESCRIPTION
**Pull request by pnlph**
in README.Data.Nat

### CODE REVIEW

File: README/Data/Nat.agda [hunk 1]
```diff
@@ -12,31 +12,34 @@ module README.Data.Nat where
...
 ex₂ = refl
 
 -- Data.Nat.Properties contains a number of properties about natural
 -- numbers.
 
-import Data.Nat.Properties as Nat
+import Data.Nat.Properties as Nat using (*-comm; +-identityʳ)
 
 ex₃ : ∀ m n → m * n ≡ n * m
 ex₃ m n = Nat.*-comm m n
```
Comment (MatthewDaggitt) [inline README/Data/Nat.agda]: Should we remove the `Nat.` qualification here then?

File: README/Data/Nat.agda [hunk 2]
```diff
@@ -12,31 +12,34 @@ module README.Data.Nat where
...
 
 -- The module ≡-Reasoning in Relation.Binary.PropositionalEquality
 -- provides some combinators for equational reasoning.
 
-open ≡-Reasoning
+open Relation.Binary.PropositionalEquality using (cong; module ≡-Reasoning)
+open ≡-Reasoning using (begin_; _≡⟨_⟩_; _∎)
```
Comment (MatthewDaggitt) [inline README/Data/Nat.agda]: Hmm I'm wondering if qualifying import statements for reasoning and solver modules is perhaps overkill? I certainly don't want to have to do so every time I use them in the main library...

File: README/Data/Nat.agda (hunk 1 above)
Comment (pnlph) [reply README/Data/Nat.agda]: Yes, because of redundancy. I did not realize.

File: README/Data/Nat.agda (hunk 2 above)
Comment (pnlph) [reply README/Data/Nat.agda]: I see you point.
It took me some time to figure out where the constructors and operators exactly come from and what do they exactly mean, so I guess for someone that is starting with the stdlib from the README files makes sense, but perhaps this must not be extensively applied to the library.

Comment (MatthewDaggitt) [reply README/Data/Nat.agda]: Okay, fair enough, that's a good argument.

File: README/Data/Nat.agda (hunk 1 above)
Comment (MatthewDaggitt) [reply README/Data/Nat.agda]: Okay, once you've removed it I'm happy and I'll merge this in.

Comment (pnlph) [reply README/Data/Nat.agda]: OK, done!

**Comment by MatthewDaggitt** [comment]
@pnlph this one has the same problem as #1009. Any chance you could revert the unneeded commits?

**Comment by pnlph** [comment]
Of course, unneeded commits reverted.

**Pull request merged**